nerfstudio/models/base_model.py

```diff
@@ -51,6 +52,11 @@ class ModelConfig(InstantiateConfig):
     eval_num_rays_per_chunk: int = 4096
     """specifies number of rays per chunk during eval"""
 
+    enable_temporal_distortion: bool = False
```

Comment:
I'm thinking we might not want to add this to the base model. If you want to add distortions to a model, one also needs to modify the model file. I worry that users may see this option in the cli and believe that enabling it will cause it to just work (but this is not the case if the model doesn't do anything with it). Thoughts on just requiring devs to add these few lines to their models if they want to use temp distortion?

Reply:
that is probably much safer in terms of user understanding and correctness, since it does need to be added per model. It would be nice to get it on all the models, but realistically probably not going to happen for a bit

Reply:
I've moved this specific config into the `VanillaConfig`, but it is still checked in the base `Model` so all models will have `temporal_distortion = None` unless it is set by a dev in the config (and presumably used in the model). The use of `getattr` might rely more on dynamic typing depending on taste, but I think it is fitting for this circumstance.

Reply:
I think this places the flag at a higher level than it should be. For example, now if you type `--help` on any model you get the following
<img width="691" alt="image" src="https://user-images.githubusercontent.com/3310961/200638326-20d3b9d8-37fd-412c-ae7e-25ff00a8df45.png">
(This is for nerfacto)

It looks like you have the flag in the VanillaModelConfig, can this one be used
<img width="562" alt="image" src="https://user-images.githubusercontent.com/3310961/200639201-6f9b824c-0c2b-4af3-9879-68256298a3f0.png">

Reply:
I think having it in `--pipeline.model.<temporal-distortion>` is where it was previously, but will try to move it deeper so it only appears for vanilla nerf.

Reply:
Currently the parameter is shown here 
![image](https://user-images.githubusercontent.com/7675847/200931086-04718501-c4d9-462f-87e8-1f18cae494d1.png)

I realize there was an additional `temporal-distortion` which was unused which was shown to all models.